src/styles/blocks/header/import_header.scss

```diff
@@ -0,0 +1,77 @@
+body {
```

Comment:
it's not part of `header` BEM block, so shouldn't be here
it should be defined in the `normalize.scss` / `reset.scss` or at least in the index.scss

Reply:
UPD: I see there are styles for the body in another file, so you probably forgot to delete it here